hmpps-kotlin-spring-boot-autoconfigure/src/test/kotlin/uk/gov/justice/hmpps/kotlin/auth/service/GlobalPrincipalOAuth2AuthorizedClientServiceTest.kt

```diff
@@ -0,0 +1,151 @@
+package uk.gov.justice.hmpps.kotlin.auth.service
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClient
+import org.springframework.security.oauth2.client.registration.ClientRegistration
+import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository
+import org.springframework.security.oauth2.core.AuthorizationGrantType
+import org.springframework.security.oauth2.core.OAuth2AccessToken
+import java.time.Duration
+import java.time.Instant
+
+@ExtendWith(MockitoExtension::class)
```

Comment:
I'm not sure this is required these days.

Reply:
I'm still getting issues using `@Mock` without the extension in place - is there something I'm missing?

Reply:
Sorry my bad - we tend to use `org.mockito.kotlin.mock` instead of annotations. All good!